### ISSUE TITLE
Did you abandoned the Videobox project?

### ISSUE THREAD

**Issue by baijianpeng** [HitkoDev/Videobox#2]
hi, Hitko,

I can not find the download link for Videobox plugin for Joomla on your website ( http://hitko.si ).

Did you abandoned this project?

If not, please tell me where can I download the latest version of this plugin.

Thank you.

**Comment by addondev** [comment]
Thank you for a very good extension, Hitko! Look forward to a new version!

**Comment by HitkoDev** [comment]
I didn't have enough time to keep updating Videobox, and there was plenty of legacy code and a mess of other things which made it outdated, and confusing for new users. In the mean time I've also switched my main development platform from Joomla! to MODX, so a Joomla! plugin wasn't my number one priority, and a whole lot of other things happened (PHP 7, Video.js 5, Node.js / Gulp /  Sass / TypeScript became a thing, HTML5 became a full standard), which all affected the development of Videobox. 

After almost two years I've finally found the time to clean-up the code, unify the development process among Videobox distributions, and make Videobox publicly available again. It's available for download from [Joomla! extensions directory](http://extensions.joomla.org/extensions/extension/multimedia/multimedia-players/videobox), [MODX extras](https://modx.com/extras/package/videobox), [my website](https://hitko.eu/videobox/download/), and from the GitHub repositories.

Unfortunately I don't have much time to work on projects like these, but hopefully the new and reorganised structure will make it easier to develop and maintain. That being said, I'm sorry everyone had to wait this long for an update.
